ibmcloud/image/build.sh

```diff
@@ -53,7 +66,13 @@ fi
 
 function cleanup () {
     msg=$1
-
+    if [ "${SE_BOOT}" = "1" ]; then
+        for mnt in "$dst_mnt/boot-se" "$dst_mnt/etc/keys" "$dst_mnt/sys"; do
+            mountpoint -q "$mnt" && umount "$mnt"
+            [[ -d "$mnt" ]] && rmdir "$mnt" 2> /dev/null || true
+        done
+        cryptsetup close $LUKS_NAME 2> /dev/null || true
```

Comment:
I think this would be an unnecessary change, the `|| true` means even if the command fails it won't affect further execution.

Reply:
I think the addition of: `set -o errexit -o pipefail -o nounset` however means this command will fail on `LUKS_NAME: unbound variable`. Rather than avoiding the error.

Reply:
Ah you're right `-o nounset` causes exit even when ignoring errors. Thanks